archive/100-theorems-list/inverse-triangle-sum.lean

```diff
@@ -0,0 +1,87 @@
+import tactic
+import algebra.big_operators
+import data.real.basic
+
+lemma range_insert (n : ℕ) :
+insert n (finset.range n) = finset.range (n + 1) :=
+begin
+ext, rw finset.mem_range,
```

Comment:
Our style is to add a level of indentation inside `begin` / `end` blocks: https://leanprover-community.github.io/contribute/style.html

Reply:
Whoops, I missed Mario's review which said the same thing.